src/api.js

```diff
@@ -33,3 +34,7 @@ export const voteOnSingleReview = (review_id, increment) => {
       throw err;
```

Comment:
Just a heads up, we shouldn't have any catch blocks or errors thrown in the api file. Your back end should be taking care of that.